library/NotificationCenter/MessageDraft/EmailMessageDraft.php

```diff
@@ -170,6 +170,14 @@ public function getHtmlBody()
     }
 
     /**
+     * Returns true if images should not be embedded
+     * @return boolean
+     */
+    public function useExternalImages() {
+        return !!$this->objLanguage->email_external_images;
```

Comment:
Just push the changes to the branch ('optional-image-embedding') of your fork, the PR will automatically get updated ;-)